unlock-app/src/components/content/CheckoutContent.tsx

```diff
@@ -43,20 +40,10 @@ export const CheckoutContent = ({
       {account && (
         <BrowserOnly>
           <Account network={network} account={account} />
-          {loading && (
-            <div>
-              {lockAddresses.map(address => (
-                <LoadingLock key={address} />
-              ))}
-            </div>
-          )}
-          {locks && (
-            <div>
-              {locks.map(lock => (
-                <Lock lock={lock} key={lock.name} />
-              ))}
-            </div>
-          )}
+          <Locks
```

Comment:
Compartmentalizing this piece inside an area where we know we always have an account is important since the hooks rely on the account address to get balance.